CHANGELOG.md

```diff
@@ -35,11 +35,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   for consistency, and updated references accordingly.
 - Changed file input extension handling to avoid hardcoding extension values.
 - Changed the variants of `Column` to use structs instead of tuples.
-- Removed `br_separator` for `Column::{Checkbox | Radio}`.
-  - `Column::{Checkbox | Radio}::display` is now injected directly into the HTML
-    stream using `Html::from_html_unchecked` of Yew.
 - Changed styling for `user_input_composite.rs` and `user_input.rs` to enahnce
   design and funcionality for `view_group`.
+- Removed `br_separator` for `Column::{Checkbox | Radio}`.
+- The items of `Column::{Checkbox | Radio}::{display | modal}` are now injected
```

Comment:
Since `modal` is newly introduced, while `display` already exists, the introduction of `modal` should be mentioned before this line. I suggest the following:

```markdown
- Introduced `Column::{Checkbox | Radio}::modal`, which displays a modal window
  corresponding to each of the `Column::{Checkbox | Radio}::display` elements.
  - The content of both `modal` and `display` is directly injected
    into the HTML stream using `Html::from_html_unchecked` from Yew.
```

Reply:
Done